src/DataIndex/Filter/Asset/MetaData/IsAssetFilterTrait.php

```diff
@@ -22,7 +22,7 @@
 /**
  * @internal
  */
-trait IsAssetMetaDataTrait
+trait IsAssetFilterTrait
```

Comment:
Not sure if its just displayed like this in github, but should we move this trait out of the MetaData namespace?

Reply:
Good Point.. I will move it :)